.github/scripts/config.sh

```diff
@@ -12,7 +12,7 @@ CYGWIN_PACKAGES_BRANCH=${CYGWIN_PACKAGES_BRANCH:-main}
 COCOM_BRANCH=${COCOM_BRANCH:-master}
 
 ARCH=${ARCH:-aarch64}
-PLATFORM=${PLATFORM:-w64-mingw32}
+PLATFORM=${PLATFORM:-pc-cygwin}
```

Comment:
All changes in this file are here just for the convenience of the local development and  will be reverted before merge.